evolver/app/main.py

```diff
@@ -128,6 +128,16 @@ async def calibrate(name: str, data: dict = None):
     return calibrator.run_calibration_procedure(data)  # TODO: or **data?
 
 
+@app.post("/abort")
+async def abort():
+    app.state.evolver.abort()
+    # Disable commit also in persistent config in case application needs to restart
+    config = Evolver.Config.load(app_settings.CONFIG_FILE)
+    config.enable_control = False
+    config.enable_commit = False
+    config.save(app_settings.CONFIG_FILE)
```

Comment:
Does this mean that whist the user can hit this endpoint to stop the control, they have to use the ``update`` endpoint to start/restart it and have to specifically do so by (hopefully) only manually changing these two flags to ``True`` every time ?

Reply:
yes!

Reply:
I don't think this is a sufficient user interface and have opened #198 to add a "start" endpoint.